apps/api/src/app/workflows-v2/workflow.controller.ts

```diff
@@ -161,6 +164,23 @@ export class WorkflowController {
     );
   }
 
+  @Get('/:workflowId/step/:stepUuid/metadata')
```

Comment:
Folks, I am sorry to be the party pooper but please remove the metadata please. This is the Get Step endpoint.

If not mistaken instead of `stepUuid`, we must use `stepIdOrStepUuid` to make this more versatile for the FE.

Reply:
it is not a get step endpoint, it's not returning the step resource, it's returning metadata required to display a step editor.
the variables constructed there does not belog on the step

Reply:
For posterity, this was resolved offline.